glTF-BinExporter/glTF/GlBuffer.cs

```diff
@@ -75,6 +76,22 @@ public void Add(Point3d point)
             PrimitiveCount += 1;
         }
 
+        public void Add(Vector3d point)
+        {
+            // Switch GL coords for Y<=>Z
+            float[] coords = new float[] { (float)point.X, (float)point.Z, -(float)point.Y };
+            Add(coords);
+            PrimitiveCount += 1;
+        }
+
+        public void Add(Point2f point)
```

Comment:
@jrz371 I went with `Point2f` because that was the type of the texcoord. But honestly don't know if it should be `Point2d` instead, and didn't lookup the difference. Any opinion?

Reply:
Point2f is single precision and Point2d is double. glTF only supports single precision so there's nothing to worry about here.